app/controllers/application_controller.rb

```diff
@@ -1,2 +1,20 @@
 class ApplicationController < ActionController::API
+  INCOGNIA_INSTALLATION_ID_HEADER = 'Incognia-Installation-ID'.freeze
```

Comment:
It is used at Sessions and Signups controllers, so I extracted it from SignupsController.

I don't see a significant difference between this constant and a private method. Actually, I prefer a constant instead of a private method inherited. :thinking: 

I don't think this is the best place, but I also don't see a problem with doing this way for now.

Reply:
The idea of having a private method that returns the installation id in ApplicationController was to have the implementation of retrieving the installation from request headers in the class where the constant is defined.

But OK, is not a big deal.

Reply:
Got it! :eyes: 

Considering it is a one-line logic that is used only at these two controllers, I'm gonna keep it this way. But of course, we can change our minds at some point in the future.